users/tokens.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils import six
+
+
+class TokenGenerator(PasswordResetTokenGenerator):
+    def _make_hash_value(self, user, timestamp):
+        return six.text_type(user.pk) + six.text_type(timestamp) + six.text_type(user.is_verified)
```

Comment:
We shouldn't need any `six` stuff. This was used as a shim between python 2 and python 3. If you just cast to `str` then this should work fine ([see how `text_type` is defined here](https://docs.djangoproject.com/en/1.11/_modules/django/utils/six/)).

Reply:
Thank you! I'll take a look at that. The examples I found for this probably came from an older django verison.